Editor/Scripts/wind_forces_tutorial.py

```diff
@@ -0,0 +1,94 @@
+"""
+Copyright (c) Contributors to the Open 3D Engine Project.
+For complete copyright and license terms please see the LICENSE at the root of this distribution.
+
+SPDX-License-Identifier: Apache-2.0 OR MIT
+"""
+
+import sys
+import azlmbr
+
+from PySide2.QtWidgets import QMenuBar
+
+from tutorial import Tutorial, TutorialStep
+
+class WindForcesTutorial(Tutorial):
+    def __init__(self):
+        super(WindForcesTutorial, self).__init__()
+
+        self.title = "PhysX Wind Forces"
+
+        self.add_step(TutorialStep("Create Wind Forces", 
+                """<html><p style="font-size:13px">Greetings!<br><br>A <i>wind provider</i> is an entity that 
+                defines a global or localized wind force that can affect certain components such as <b>Cloth</b> components. 
+                <br><br>In this tutorial, we'll set up a local wind provider, add a cloth object to the scene, and configure the cloth object to simulate a sheet gently blowing in the breeze. 
+                Let's begin by creating a new entity for a wind provider. Right click in <b>Entity Outliner</b> and choose <b>Create Entity</b> from the context menu. Name the new entity 
+                <strong><code style="font-size:14px;color:#E44C9A">"Wind Provider"</code></strong>.
+                <br><br>For this tutorial, make sure that you have the <b>Nvidia Cloth Gem</b> and <b>PhysX Gem</b> enabled in your project.</p></html>"""))
+        self.add_step(TutorialStep("Add a Tag component",  """<html><p style="font-size:13px">
+                We need to add a tag to the entity to identify it as a wind provider and specify the type of wind force. We can do this with a Tag component.
+                In Entity Inspector, choose Add Component and select Tag from the components list to add a Tag component to the entity. </p></html>""", "InspectorMainWindow"))
```

Comment:
Slight modification of the explanation. Bold UI components. Highlight the Add Component button.

```suggestion
        self.add_step(TutorialStep("Add a Tag component",  """<html><p style=
                "font-size:13px">The entity needs a tag that identifies it as a 
                wind provider and specifies the type of wind force. You can add 
                the tag with a <b>Tag</b> component.<br><br>In Entity Inspector,
                 choose <b>Add Component</b> and select <b>Tag</b> from the 
                components list to add a Tag component to the entity. 
                </p></html>""", "m_addComponentButton"))
```